vms/avm/environment_test.go

```diff
@@ -524,7 +524,9 @@ func issueAndAccept(
 	issuer <-chan common.Message,
 	tx *txs.Tx,
 ) {
-	txID, err := vm.IssueTx(tx.Bytes())
+	vm.ctx.Lock.Unlock()
+	txID, err := vm.issueTx(tx)
+	vm.ctx.Lock.Lock()
```

Comment:
Why can't we modify the tests calling this to just not hold the lock?

Reply:
I'll make a PR on top of this PR and we can decide which we prefer

Reply:
https://github.com/ava-labs/avalanchego/pull/2526